src/modules/dashboard/components/Vesting/VestingPageLayout.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { MessageDescriptor, defineMessage } from 'react-intl';
+import Button from '~core/Button';
+
+import Numeral from '~core/Numeral';
+import { SpinnerLoader } from '~core/Preloaders';
+import { SimpleMessageValues } from '~types/index';
+import { getFormattedTokenValue } from '~utils/tokens';
+
+import styles from './Vesting.css';
+
+const MSG = defineMessage({
+  loading: {
+    id: 'dashboard.Vesting.VestingPageLayout.loading',
+    defaultMessage: 'Loading',
+  },
+});
+
+const displayName = 'dashboard.Vesting.VestingPageLayout';
+
+interface Props {
+  title: JSX.Element;
+  buttonText: MessageDescriptor | string;
+  buttonTextValues?: SimpleMessageValues;
+  tableValues: {
+    label: JSX.Element;
+    value: string;
+  }[];
+  tokenDecimals: number;
+  isLoading: boolean;
+}
+
+const VestingPageLayout = ({
+  title,
+  tableValues,
+  buttonText,
+  tokenDecimals,
+  buttonTextValues,
+  isLoading,
+}: Props) => {
```

Comment:
How will this component handle the different sagas that we need to send ? (one for unwrapping and one for claiming)

My point is that the Button should not be contained in here, but should be added individually to each component, as to be wrapped in a different `ActionForm`

Reply:
I've added the ActionForm to both pages and I think it doesn't require to have Button component twice. Please, check my implementation and let me know if you have something different in mind.
For now I've addded just Generic actions taking into account that you will change them once you wire this stuff up. But let me know if you want me to add already actual actions.